core/src/main/scala/kafka/tools/StorageTool.scala

```diff
@@ -311,7 +311,7 @@ object StorageTool extends Logging {
     formatParser.addArgument("--release-version", "-r")
       .action(store())
       .help(s"The release version to use for the initial feature settings. The minimum is " +
-        s"${MetadataVersion.MINIMUM_VERSION}; the default is ${MetadataVersion.LATEST_PRODUCTION}")
+        s"${MetadataVersion.MINIMUM_VERSION}; the default is ${MetadataVersion.latestTesting().version()}")
```

Comment:
Using `latestTesting` as default version is a bit weird to me. Could you please share more details with me?

Reply:
Yes, you're right, the default version should not be modified, so I’ve reverted this change.